tests/034_onload.ts

```diff
@@ -0,0 +1,4 @@
+window.onload = function(e): void {
+  console.log(`got ${e.type} event`);
+};
+console.log("main");
```

Comment:
Might be good idea to add 1-2 imports in this file, that do `console.log`  to ensure that `onload` is actually called after imports are resolved. WDYT?

Reply:
Agree. I added an import statement.